github/repos.go

```diff
@@ -529,6 +531,12 @@ type RequiredStatusChecks struct {
 	Contexts *[]string `json:"contexts,omitempty"`
 }
 
+// RequiredPullRequestReviews represents the protection configuration for pull requests.
+type RequiredPullRequestReviews struct {
+	// Enforce pull request reviews for repository administrators.
+	IncludeAdmins *bool `json:"include_admins,omitempty"`
```

Comment:
Thanks for bringing this up.

I was mostly following what I thought was a convention. As another example, something like [creating a repository](https://developer.github.com/v3/repos/#create) requires a `name` in the same manner, but [`name` is `omitempty` on the `Repository` struct](https://github.com/google/go-github/blob/dcda0b96bd591fbeb40dcaddec3fb40a7fcaca34/github/repos.go#L23). It seems this pattern is pretty common in this codebase.

I also initially thought that in the absence of `omitempty`, failing to provide a value would send `null` along which might evaluate to `false` server-side, and therefore be surprising. However, on actually testing this assumption, it appears that GitHub validates that `include_admins` is strictly a boolean and will raise an error if `null` is passed. The assumption was wrong.

If you think that going forward it's a good idea for required values to, well, omit `omitempty`, I'm on board and will make it so for this property. Let me know what you think.